cypress/package.json

```diff
@@ -11,6 +11,6 @@
   },
   "scripts": {
     "cypress:open": "cypress open",
-    "cy:run": "cypress run --config defaultCommandTimeout=15000"
+    "cy:run": "cypress run --browser chrome --config defaultCommandTimeout=15000"
```

Comment:
Currently, there is an issue with Electron, see https://github.com/cypress-io/cypress/issues/8865

Reply:
Is there any change required to be able to run on windows or?

Reply:
I think it should be fine, if you have issues with Electron throwing the mentioned error you can just use Edge or Chrome or ignore it, it's more relevant in CIs as it could block merging the PRs.